docs/docker/docker-compose-browserless.yaml

```diff
@@ -8,10 +8,9 @@
 # To start the containers, run:
 #   docker-compose -f docs/docker/docker-compose-browserless.yaml up
 
-version: '3.9'
-
 services:
   big-agi:
+  container_name: big-agi
```

Comment:
I have a lint error here, because the indent is the same as the `big-agi:` and should be more?